pkg/random/random_test.go

```diff
@@ -99,16 +99,48 @@ func TestPseudoGenerators(t *testing.T) {
 	}
 }
 
```

Comment:
Such tests do not have much added value. Instead of covering one random value, you should cover a range or better option would be to use fuzzy testing: https://go.dev/doc/fuzz/

Reply:
Thanks for the link in the comment, I will investigate and fix it.